src/components/com_tjcertificate/administrator/languages/en-GB/en-GB.com_tjcertificate.ini

```diff
@@ -286,3 +286,26 @@ COM_TJCERTIFICATE_IMAGE_GENERATION_TYPE_DESC="There are two ways certificate ima
 COM_TJCERTIFICATE_IMAGE_GENERATION_TYPE_CANVAS="HTML2Canvas Library"
 COM_TJCERTIFICATE_IMAGE_GENERATION_TYPE_IMAGICK="PHP ImageMagick Library"
 COM_TJCERTIFICATE_IMAGE_GENERATION_SETTINGS_NOTE="Please note: <br> <br> <label>You need to follow below steps to make it work properly with <i>PHP ImageMagick Library</i>: <br><br> 1. PHP's ImageMagick extension must be installed in your server <br> 2. We are generating image using PDF, but by default ImageMagick does not allow reading/writing PDF files, you need to manually change the settings for PDF option in <i>/etc/ImageMagick-6/policy.xml</i> file for Ubuntu/Linux OS. For Windows OS, the path will be different <br> 3. By default the PDF setting looks like <i>&lt;policy domain='coder' rights='none' pattern='PDF' /&gt;</i> <br> 4. After changes, it should look like <i>&lt;policy domain='coder' rights='read|write' pattern='PDF' /&gt;</i> <br> 5. Once done with above steps, restart PHP & web server(Apache/Nginx etc), it should work now</label>"
+
+;__DEPLOY_VERSION__
+;Permission for external record
+COM_TJCERTIFICATE_CLIENT_EXTERNAL="External"
+COM_TJCERTIFICATE_PERMISSION_CREATE_EXTERNAL_CERTIFICATE="Create external certificate"
+COM_TJCERTIFICATE_PERMISSION_CREATE_EXTERNAL_CERTIFICATE_DESC="Allow users to create external certificate"
+COM_TJCERTIFICATE_PERMISSION_MANAGE_EXTERNAL_CERTIFICATE="Manage external certificate"
+COM_TJCERTIFICATE_PERMISSION_MANAGE_EXTERNAL_CERTIFICATE_DESC="Allow users to manage external certififcate"
+COM_TJCERTIFICATE_PERMISSION_MANAGEOWN_EXTERNAL_CERTIFICATE="Manage own external certificate"
+COM_TJCERTIFICATE_PERMISSION_MANAGEOWN_EXTERNAL_CERTIFICATE_DESC="Allow users to manage own external certififcate"
```

Comment:
Update from line 292 to 298. Replace last word `certificate` with `certificates`

Reply:
done

Reply:
I can't see your changes

Reply:
??

Reply:
updated to records